src/containers/ArticlePage/components/ResourceTypeSelect.jsx

```diff
@@ -10,9 +10,6 @@ import { FieldHeader, Select } from '@ndla/forms';
 import { injectT } from '@ndla/i18n';
 import { selectedResourceTypeValue } from '../../../util/taxonomyHelpers';
 import HowToHelper from '../../../components/HowTo/HowToHelper';
-import { RESOURCE_TYPE_LEARNING_PATH } from '../../../constants';
-
-const blacklistedResourceTypes = [RESOURCE_TYPE_LEARNING_PATH];
```

Comment:
I denne fila må vel blacklisted resource types finnes, for denne brukes fra learningresourcepage.